backend/tests/test_media_handler.py

```diff
@@ -0,0 +1,93 @@
+import tempfile
+import unittest
+from unittest.mock import patch, MagicMock
+from pathlib import Path
+from backend.media_handler import MediaHandler
+
+class TestMediaHandler(unittest.TestCase):
+
+    def setUp(self):
+        """Set up base directory and mock paths for testing."""
+        self.base_directory = tempfile.mkdtemp()
+        self.video_url = "https://www.youtube.com/watch?v=dQw4w9WgXcQ"
```

Comment:
Please don't place data inside of the code. That later would require code updates for simple test data changes. These should be placed in appropriate test data files. You can see what has been done for the MediaProcessor for examples.